action.yaml

```diff
@@ -75,12 +75,20 @@ inputs:
     description: 'Browser type for UI tests'
     required: false
     default: ''
+  app-id:
+    required: false
+    description: 'APP ID'
+    default: ''
+  ta-upgrade-version:
+    required: false
+    description: 'upgrade version of TA'
```

Comment:
Shouldn't we make it clearer that this is the "pre-upgrade" version to run with? Or is it the other way around?